media/css/foundation/foundation.less

```diff
@@ -56,6 +56,23 @@ img {
   height: auto;
 }
 
+ol {
+  ul {
+    list-style-type: disc;
+  }
+}
+
+ul + p {
+  margin-top: @baseLine;
+}
+
+.example-link {
+  &:after {
```

Comment:
We have a standard `.trailing-arrow` mixin for this, if that makes things easier.